src/py21cmfast/__init__.py

```diff
@@ -18,21 +18,11 @@
 from os import mkdir as _mkdir
 from os import path
 
-from . import cache_tools, inputs, lightcones, outputs, plotting, wrapper
+from . import cache_tools, lightcones, plotting, wrapper
 from ._cfg import config
 from ._logging import configure_logging
 from .cache_tools import query_cache
 from .lightcones import AngularLightconer, RectilinearLightconer
-from .outputs import (
-    Coeval,
-    HaloBox,
-    InitialConditions,
-    IonizedBox,
-    LightCone,
-    PerturbedField,
-    TsBox,
-    XraySourceBox,
-)
 from .wrapper import (
```

Comment:
I'm guessing we need to move most of these to their respective submodules?

Reply:
Would we prefer to access inputs and outputs at the upper level i.e `py21cmfast.BrightnessTemperatureBox` or `py21cmfast.outputs.BrightnessTemperatureBox`

Reply:
I think at the upper level, so they should be imported at the `__init__`.